app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,28 @@
+<footer>
+  <div class='wrapper'>
+      <div class='col-1 first'>
+        <h3>Company</h3>
+        <ul>
+          <li><%= link_to "About Us", arts_path %></li>
+          <li><%= link_to "Contact Us", arts_path %></li>
+        </ul>
+      </div>
+      <div class='col-1'>
+        <h3>Discover</h3>
+        <ul>
+          <li><%= link_to "Why chose us", arts_path %></li>
+          <li><%= link_to "Our Art Pieces", arts_path %></li>
+          <li><%= link_to "Top Rated", arts_path %></li>
+          <li><%= link_to "Client reviews", arts_path %></li>
+        </ul>
+      </div>
+      <div class='col-1'>
+        <h3>Rent your Art</h3>
+        <ul>
+          <li><%= link_to "Why Rent", arts_path %></li>
+          <li><%= link_to "Responsable Renting", arts_path %></li>
+          <li><%= link_to "Instant Book", arts_path %></li>
+        </ul>
+      </div>
+  </div>
+</footer>
```

Comment:
Can you give this a bit of spacing to make it easier to read ?

We don't have that many pages do we ? There's no point in adding stuff to a footer just for decoration.
A footer can be just as smll as this forexample
![image](https://user-images.githubusercontent.com/75388869/220066484-809baa44-093b-4dca-a504-b315ee897ee1.png)

Or even smaller. If we have nothing to say in the navbar then you can just say "made by ..." and share the github repo